automated/linux/kselftest/kselftest.yaml

```diff
@@ -17,8 +17,8 @@ metadata:
 params:
     # Specify Linux kernel self test pre-build tarball name from below location
     # http://testdata.validation.linaro.org/tests/kselftest/
-    # TESTPROG: kselftest_aarch64.tar.gz
-    TESTPROG: kselftest_armhf.tar.gz
+    TESTPROG: kselftest_aarch64.tar.gz
```

Comment:
is there a reason to use aarch64 as default? This can be overwritten in the LAVA job definition.